tests-ui/tests/components/templates/templateWorkflowCard.test.ts

```diff
@@ -0,0 +1,130 @@
+import { shallowMount } from '@vue/test-utils'
+import { describe, expect, it } from 'vitest'
+
+import TemplateWorkflowCard from '@/components/templates/TemplateWorkflowCard.vue'
+
+// import { TemplateInfo } from '@/types/workflowTemplateTypes'
+
+describe('correctThumbnailVariant', () => {
+  it('core default variant', async () => {
+    const data = {
+      sourceModule: 'default',
+      categoryTitle: '',
+      loading: false,
+      template: {
+        name: 'test',
+        description: 'test',
+        mediaType: 'image',
+        mediaSubtype: 'webp'
+      }
+    }
+    const wrapper = shallowMount(TemplateWorkflowCard, { propsData: data })
+    // @ts-expect-error 'thumbnailVariant is computed'
```

Comment:
I don't really understand why this is needed, the computed does exists. and the test works.
Any idea how to properly fix this ?